src/components/App.tsx

```diff
@@ -270,19 +271,18 @@ export class App extends React.Component<any, AppState> {
             iv,
           );
 
+          let deletedIds = this.state.deletedIds;
```

Comment:
Can you put it inside of the right case: 

```js
case "SCENE_UPDATE":
```

Reply:
Cannot as this needs to be accessible from the outer scope because we're using it in the `setState`.

Reply:
Arg, I checked the indentation level and made a mistake. Sorry, disregard my comment :)